src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -1038,7 +1038,8 @@ schedule_delta_garbage_collection() ->
     end.
 
 %% @private
-send(Msg, Peer) ->
+send({Type, _From, Payload}=Msg, Peer) ->
```

Comment:
In addition to send, you're also going to need this instrumentation in both ```delta_send``` and ```delta_ack```.